libs/env-collector/src/lib.rs

```diff
@@ -389,7 +491,7 @@ fn _flat_json(json: &Value, prefix: &str, env_vars: &mut BTreeMap<String, String
         _ => {
             let env_var_name = prefix.to_uppercase();
             let env_var_value = match json {
-                Value::String(s) => s.to_string(),
+                Value::String(s) => format!("\"{}\"", s),
```

Comment:
well, I don't agree that we need to serialize string with quotes. because when you specify env variables you always specify strings, like `BOOL_VAR=false` and `STR_VAR=string` -- they are essentially the same when we specify them, however in docs you will show that default value is `false` for BOOL_VAR and `"string"` for STR_VAR. therefore this may add confusion and user will specify unnecessary quotes when setting a variable of type string